discovery/service.go

```diff
@@ -239,33 +270,37 @@ func (d *Discovery) Stop() {
 func (d *Discovery) networkHandler() {
 	defer d.wg.Done()
 
-	var announcementBatch []*networkMsg
+	var announcementBatch []lnwire.Message
 
 	// TODO(roasbeef): parametrize the above
 	retransmitTimer := time.NewTicker(time.Minute * 30)
 	defer retransmitTimer.Stop()
 
 	// TODO(roasbeef): parametrize the above
-	trickleTimer := time.NewTicker(time.Millisecond * 300)
+	trickleTimer := time.NewTicker(d.cfg.TrickleDelay)
 	defer trickleTimer.Stop()
 
+	// TODO(roasbeef): parametrize the above
+	reprocessTimer := time.NewTicker(time.Second)
+	defer reprocessTimer.Stop()
+
 	for {
 		select {
 		case announcement := <-d.networkMsgs:
 			// Process the network announcement to determine if
 			// this is either a new announcement from our PoV or an
-			// updates to a prior vertex/edge we previously
-			// accepted.
-			accepted := d.processNetworkAnnouncement(announcement)
-
-			// If the updates was accepted, then add it to our next
-			// announcement batch to be broadcast once the trickle
-			// timer ticks gain.
-			if accepted {
+			// edges to a prior vertex/edge we previously
+			// proceeded.
```

Comment:
proceeded -> processed 

there're a few other occurrences in this PR that should be fixed.